cosipy/image_deconvolution/RichardsonLucy.py

```diff
@@ -225,38 +249,41 @@ def finalization(self):
         """
         finalization after running the image deconvolution
         """
```

Comment:
The same as before. Can we add these lines at the beginning instead?
```
if self.parallel_computation == True and self.master_node == False:
    pass
```

Reply:
Changes accepted.